src/pages/billing/Project.js

```diff
@@ -195,6 +195,7 @@ const groupByBillingAccountStatus = (billingProject, workspaces) => {
 const LazyChart = lazy(() => import('src/components/Chart'))
 const maxWorkspacesInChart = 10
 const spendReportKey = 'spend report'
+const otherCostsDomId = 'billing-other-costs'
```

Comment:
This approach will work, and we should never have multiple of these sections on the page. However, the general pattern in the repo is to use `IdContainer` to generate the ID and obtain the reference to it. For example, on line 602.

Reply:
removed from newer implementation